frontend/src/pages/org/settings/components/profile.ts

```diff
@@ -80,6 +78,12 @@ export class OrgSettingsProfile extends BtrixElement {
         `,
         html`
           ${msg(
+            // TODO (emma) link "update your Org URL..." to scroll to the relevant setting
+            // Unfortunately this isn't as simple as just using an id and an anchor link,
+            // as anchor links don't penetrate the shadow dom. Instead we'd most likely
+            // need to set up a global shared re-implementation of the id system, with
+            // a globally-shared map with `@lit/context` or `lit-shared-state`. Another
+            // small frustration about web components!
```

Comment:
@emma-sg This could be a link to `#org-url` and `btrix-org-settings` could scroll the correct section on a hash change event, or did you have something else in mind?